### PR TITLE
feat: 29829 allow different tag`s facilities in one model.

### PR DESCRIPTION
**Pull request by devProdigy**
## Description

Now in one models we might have tags with different facilities (got the common one as before).

* update requirements;

[Azure DevOps Link](https://dev.azure.com/EquinorASA/OMNIA%20Prevent/_workitems/edit/29829)

## Type of change

Mark the relevant options (`[x]` will mark checkbox as checked).
- [ ] Bugfix
- [X] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Tests
- [ ] Other

# Checklist:

- [ ] I have made corresponding changes to the documentation / README (if needed)
- [X] I have added tests that prove my fix is effective or that my feature works (if needed)

### CODE REVIEW

File: app/latigo/gordo/misc.py [hunk 1]
```diff
@@ -67,3 +70,26 @@ def print_client_debug(client: Optional[Client]):
...
+    Returns:
+        tag`s facility.
+    """
+    letters_to_cut_from_asset = 4
+
+    if not tag_name and not sensor_tag:
+        raise ValueError("'tag_name' or 'sensor_tag' should be populated.")
+
+    sensor_tag = sensor_tag if sensor_tag else normalize_sensor_tags([tag_name])[0]
+    return sensor_tag.asset[:letters_to_cut_from_asset]
```
Comment (koropets) [inline app/latigo/gordo/misc.py:95]: A better solution here would be taken the first numbers from `asset` and if there are no numbers there just keeping `asset` as it is

Comment (devProdigy) [reply app/latigo/gordo/misc.py:95]: is there such possibility that facility won't has numbers? (like 1000)
I thought that asset will always start from 4 numbers - facility.

Comment (koropets) [reply app/latigo/gordo/misc.py:95]: No, but it might be in the future

Comment (koropets) [reply app/latigo/gordo/misc.py:95]: For our future clients

Comment (devProdigy) [reply app/latigo/gordo/misc.py:95]: I offer not to rely on it now, cause we don't know how it'll be and such change should be added explicitly and such facilities should supported by TS API (or any other provider). So it probably be more complex than just trying to get any with number.

**Review by rkicz — APPROVED** [review]

**Review by RollerKnobster — APPROVED** [review]

**Pull request merged**